gui-apps/foot/foot-1.12.1.ebuild

```diff
@@ -0,0 +1,71 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit meson xdg systemd
+
+DESCRIPTION="Fast, lightweight and minimalistic Wayland terminal emulator"
+HOMEPAGE="https://codeberg.org/dnkl/foot"
+SRC_URI="https://codeberg.org/dnkl/foot/archive/${PV}.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}/${PN}"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~arm64"
+IUSE="+grapheme-clustering"
+
+COMMON_DEPEND="
+	dev-libs/wayland
+	media-libs/fcft
+	media-libs/fontconfig
+	x11-libs/libxkbcommon
+	x11-libs/pixman
+	grapheme-clustering? (
+		dev-libs/libutf8proc:=
+		media-libs/fcft[harfbuzz]
+	)
+"
+DEPEND="
+	${COMMON_DEPEND}
+	dev-libs/wayland-protocols
+	dev-libs/tllist
+"
+RDEPEND="
+	${COMMON_DEPEND}
+	|| (
+		>=sys-libs/ncurses-6.3[-minimal]
+		~gui-apps/foot-terminfo-${PV}
+	)
+"
+BDEPEND="
+	dev-util/wayland-scanner
+	app-text/scdoc
+"
+
+src_prepare() {
+	default
+	# disable the systemd dep, we install the unit file manually
+	sed -i "s/systemd', required: false)$/', required: false)/" "${S}"/meson.build || die
+}
+
+src_configure() {
+	local emesonargs=(
+		$(meson_feature grapheme-clustering)
+		-Dthemes=true
+		-Dime=true
+		-Dterminfo=disabled
+		-Dwerror=false
+	)
+	meson_src_configure
+
+	sed 's|@bindir@|/usr/bin|g' "${S}/"/foot-server@.service.in > foot-server@.service
+}
+
+src_install() {
+	local DOCS=( CHANGELOG.md README.md )
+	meson_src_install
+
+	rm -r "${ED}/usr/share/doc/${PN}" || die
```

Comment:
Is this empty? I see `local DOCS=( CHANGELOG.md README.md )` are being installed to right path, especially since upstream seems to have docs/ directory with content. Maybe `mv` instead if they get properly built? 

(either way if you keep this, please add a comment above why this is needed)

Reply:
foot unconditionally installs the changelog, license and readme to `/usr/share/doc/foot/`; the upstream `doc` directory are scdoc sources for manuals for the most part.